packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx

```diff
@@ -113,18 +118,13 @@ export const MetricsTable = () => {
         mantinePaperProps: {
             shadow: undefined,
         },
-        mantineTableContainerProps: {
-            ref: tableContainerRef,
-            sx: { maxHeight: '600px', minHeight: '600px' },
-            onScroll: (event: UIEvent<HTMLDivElement>) =>
-                fetchMoreOnBottomReached(event.target as HTMLDivElement),
-        },
         mantineTableProps: {
             highlightOnHover: true,
             withColumnBorders: true,
         },
         mantineTableHeadRowProps: {
             sx: {
+                paddingRight: shouldPadHeader ? '15px' : '0px', // Add padding to the right of the header to account for the scrollbar
```

Comment:
I'm not sure if this is the right approach as I don't see it scale tbh...

I understand this solution brings the quick win but maybe we should:
- see how to fix the padding progamatically 
- not go ahead with this if too much work

<img width="1391" alt="image" src="https://github.com/user-attachments/assets/d03ac735-3b6a-4d02-af01-04fef9ab69ae">


cc @PriPatel as we were also discussing scrolling behaviours.

Reply:
(ref: https://github.com/KevinVandy/mantine-react-table/issues/331#issuecomment-2131242703)